test/javascripts/initializers_test.coffee

```diff
@@ -106,10 +112,11 @@ describe 'Initializers', ->
         httpUrl: "somewhere"
         fullRefresh: false
         refreshOnSuccess: "foo"
+        refreshOnSuccessExcept: null
         refreshOnError: null
         refreshOnErrorExcept: null
 
-    it 'passes through null for missing full-refresh-on-error-except', ->
+    it 'respects full-refresh-on-error-except', ->
```

Comment:
@qq99 I think it makes sense to stay consistent with the error case logic. I've updated the code and adjusted the tests slightly.